Orleans.Sagas/SagaGrain.cs

```diff
@@ -56,7 +82,7 @@ public Task<SagaStatus> GetStatus()
             return Task.FromResult(State.Status);
         }
 
-        public async Task<IReadOnlyDictionary<Type, SagaError>> GetSagaErrors() => await Task.FromResult(State.Errors);

```

Comment:
Is the intention now to throw an exception whenever this is called? If so should we not just remove the method and the interface def?

Reply:
The reason that I kept it is that I will next PR, to store the exception that been thrown by activity using the propertybag, so it will be backward compatiable.

But I don't mind to delete them now, and added them later on.

WDYT @creyke ?

Reply:
Makes sense. Worth adding an `[Obsolete]` attribute to the interface and the implementation so we know this is the intent?